README.md

```diff
@@ -0,0 +1,67 @@
+# atoum standard edition
+
+atoum standard edition is a version of atoum that directly contains code completion in IDE, configuration via a YAML file and environnement variables to simplfy the usage of atoum on CI tools. Il also comes with some usefull extensions to test protected method or to add new reports to atoum.
```

Comment:
atoum standard edition (SE) is a version of atoum that contains atoum and some extensions. I simplified the way you can work with atoum by giving you a pref-defined set of tools.

Better?